streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -290,11 +291,40 @@ private void shutdown() {
             log.error("stream-thread [{}] Failed to close restore consumer: ", this.getName(), e);
         }
 
+        // remove all tasks
         removeStreamTasks();
+        removeStandbyTasks();
```

Comment:
Since we close tasks already, we only need to clear the tasks map in this function.